internal/statestore/backfill_test.go

```diff
@@ -604,3 +604,57 @@ func generateBackfills(ctx context.Context, t *testing.T, service Service, amoun
 
 	return backfills
 }
+
+func TestCleanupBackfills(t *testing.T) {
+	cfg, closer := createRedis(t, false, "")
+	defer closer()
+	service := New(cfg)
+	require.NotNil(t, service)
+	defer service.Close()
+	ctx := utilTesting.NewContext(t)
+
+	rc, err := redis.Dial("tcp", fmt.Sprintf("%s:%s", cfg.GetString("redis.hostname"), cfg.GetString("redis.port")))
+	require.NoError(t, err)
+
+	bfID := "mockBackfill-1"
+	ticket1ID := "t1"
+	ticket2ID := "t2"
+	ticketIDs := []string{ticket1ID, ticket2ID}
+	bfLastAck := "backfill_last_ack_time"
+	proposedTicketIDs := "proposed_ticket_ids"
+
+	// ARRANGE
```

Comment:
nit: inconsistent comments through the test. Prefer using complete sentences - you may just choose to describe a comment blurb for the test rather than commenting each method as the names are verbose anyways.

Reply:
"Arrange act assert" is a common pattern of writing tests and I've just made the test a bit more readable. You clearly see that `err = service.AddTicketsToPendingRelease(ctx, ticketIDs)`, for example, is a part of an ARRANGE section, which means it is used to prepare data for an actual test. Then I clearly separate a function under the test with ACT section, and so on. So I find these comments quite useful here.

Reply:
I like the test pattern itself - but did not think the comments were necessary. Comments are only useful when they add value not obvious when reading the code. This pattern seems clean enough and the comments // ARRANGE // ACT // ASSERT do not seem to add value. If anything, they do seem misplaced compared to any other comment in production code. 

Did find an interesting blog that goes through this exact pattern and why you might add comments - and why it is a bad idea. Interesting read: https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

Readability is subjective. You may leave them in place if you think it makes it more readable. It did not make it any more readable for me.